content/topics/kubernetes/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Kubernetes
 weight: 2
+featured: true
```

Comment:
Seems weird to highlight a topic page. Maybe we highlight the most recent other blog as well?

Reply:
Sure thing, let me update it with a new commit